tests/test_shed_tools_methods.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+from ephemeris.shed_tools_methods import *
+
+
+def test_flatten_repo_info():
+    test_repositories = [
```

Comment:
I'd give 💯 bonus points for also testing an invalid key. You should be able to use https://docs.pytest.org/en/latest/logging.html#caplog-fixture to assert that the correct warning is being emitted.